rust/src/pgsql/pgsql.rs

```diff
@@ -356,9 +376,22 @@ impl PgsqlState {
                         _needed,
                         needed_estimation
                     );
+                    SCLogDebug!("PgsqlEvent::TruncatedData");
+                    self.set_event(PgsqlEvent::TruncatedData);
                     return AppLayerResult::incomplete(consumed as u32, needed_estimation as u32);
                 }
+                Err(Err::Error(Error{code:ErrorKind::Verify, ..})) => {
```

Comment:
It is not obvious to me why a `verify` error is `invalidlength`

In the future, I may add more pgsql parsing code with nom `verify` that is not about lengths

Reply:
true that >_<'
I'll re-write this part.